### ISSUE TITLE
debug exception

### ISSUE THREAD

**Issue by zolotov** [ballerinalang/plugin-intellij#402]
IDEA Community 163.7743

```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.execution.process.ProcessHandler$4.invoke(ProcessHandler.java:226)
	at com.sun.proxy.$Proxy14.processTerminated(Unknown Source)
	at com.intellij.execution.process.ProcessHandler$3.run(ProcessHandler.java:183)
	at com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:259)
	at com.intellij.execution.process.ProcessHandler.notifyTerminated(ProcessHandler.java:164)
	at com.intellij.execution.process.ProcessHandler.notifyProcessDetached(ProcessHandler.java:156)
	at com.intellij.xdebugger.DefaultDebugProcessHandler.detachProcessImpl(DefaultDebugProcessHandler.java:33)
	at com.intellij.execution.process.ProcessHandler$2.run(ProcessHandler.java:125)
	at com.intellij.execution.process.ProcessHandler$TasksRunner.execute(ProcessHandler.java:259)
	at com.intellij.execution.process.ProcessHandler.detachProcess(ProcessHandler.java:120)
	at com.intellij.execution.actions.StopAction.stopProcess(StopAction.java:247)
	at com.intellij.execution.actions.StopAction.actionPerformed(StopAction.java:185)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:199)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:215)
	at com.intellij.openapi.actionSystem.impl.ActionButton.actionPerformed(ActionButton.java:188)
	at com.intellij.openapi.actionSystem.impl.ActionButton.performAction(ActionButton.java:154)
	at com.intellij.openapi.actionSystem.impl.ActionButton.processMouseEvent(ActionButton.java:325)
	at java.awt.Component.processEvent(Component.java:6300)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4891)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2750)
	at java.awt.Component.dispatchEvent(Component.java:4713)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:76)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:675)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
Caused by: java.lang.AbstractMethodError
	at com.intellij.xdebugger.XDebugProcess.stop(XDebugProcess.java:180)
	at org.ballerinalang.plugins.idea.debugger.BallerinaDebugProcess.stop(BallerinaDebugProcess.java:143)
	at com.intellij.xdebugger.XDebugProcess.stopAsync(XDebugProcess.java:185)
	at com.intellij.xdebugger.impl.XDebugSessionImpl.stopImpl(XDebugSessionImpl.java:861)
	at com.intellij.xdebugger.impl.XDebugSessionImpl.access$000(XDebugSessionImpl.java:82)
	at com.intellij.xdebugger.impl.XDebugSessionImpl$1.processTerminated(XDebugSessionImpl.java:270)
	... 53 more
```

**Comment by Shan1024** [comment]
Hi @zolotov,

I have made some changes to the debugger branch in the evening, so this issue should be fixed by now.

PS: I am a big fan of your work at **go-lang-idea-plugin** :) I used it as a reference since i could not find much documentation on developing advance features of Intellij IDEA. Awesome work (Y)

**Comment by zolotov** [comment]
Hi @Shan1024. Unfortunately, I cannot check whether it's fixed, this exception was delivered to Jetbrains's exception analyzer, so I don't know how to recreate it.

This is really cool that go-lang-idea-plugin helps you to make this plugin! Btw, there is also [Open API forum](https://intellij-support.jetbrains.com/hc/en-us/community/topics/200366979-IntelliJ-IDEA-Open-API-and-Plugin-Development), it might be useful as well.

**Comment by Shan1024** [comment]
Hi @zolotov,

I may have accidentally submitted the exception when I was testing. Sorry about that :)

I referred the forum in the past when I had issues. But it seems to be not useful sometime. I posted a [question](https://intellij-support.jetbrains.com/hc/en-us/community/posts/115000111430-Custom-Language-plugin-is-not-suggesting-for-bal-files) about 3 months ago, but still I haven't received a reply :( Still not sure how to fix this and whether this should be fixed in the jetbrains plugin repository or on the plugin side. Appreciate any insight into this.
